### DISCUSSION TITLE
How to add localisation prefix to admin uri

### DISCUSSION THREAD

**Discussion by jordantsap** [filamentphp/filament#14746] [Help]
### Package

Panel builder

### Package Version

3.2

### How can we help you?

I am building an application with spatie/laravel-translatable to which i have the $locale  $locale = request()->segment(1) embedded into the uri of the application, in the admin it gives me error 404, how to add the locale code in the admin uri path

**Comment by nhanchaukp** [comment]
Im using package `mcamara/laravel-localization` and use this setup

```php
# routes/web.php

Route::group([
    'prefix' => LaravelLocalization::setLocale(),
    'middleware' => ['localeSessionRedirect', 'localizationRedirect', 'localeViewPath', 'localize'],
], function () {
    //...
    require base_path('vendor/filament/filament/routes/web.php');
});
```

in `AdminPanelProvider.php` set `->path('')`

**Reply by ctf0** [reply]
i would recommend to keep the path so u can prefix your routes, also add 

```php
require base_path('vendor/filament/actions/routes/web.php');
```
